frontend/public/components/dashboard/grid.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from 'react';
 import { Grid, GridItem } from '@patternfly/react-core';
+import { global_breakpoint_lg as breakpointLG } from '@patternfly/react-tokens';
```

Comment:
Looking at `node_modules/@patternfly/react-tokens/dist/js/index.js` the exported objects looks like

```js
{"name":"--pf-global--breakpoint--lg","value":"992px","var":"var(--pf-global--breakpoint--lg)"}
```

so if `value` changes its CSS unit from `px` to something else our layout will probably break.. (not sure we can do anything to prevent that)

Reply:
I was looking at this too. One (small) problem I have with using these breakpoint vars is that we are using breakpoints against the width of an element (instead of the viewport). Seems a bit arbitrary, and maybe a bit confusing since it will cause reflow at different viewport widths than everything else. I don't think it's a huge deal, which is why I didn't leave a comment originally, but since it came up 😆